src/datagen/main/java/mekanism/common/recipe/impl/ControlCircuitRecipeProvider.java

```diff
@@ -44,4 +50,12 @@ private void addCircuitUpgradeRecipe(RecipeOutput consumer, IItemProvider output
               .key(Pattern.ALLOY, alloyTag)
               .build(consumer, Mekanism.rl(basePath + name));
     }
+
+    private void addCircuitInfusionUpgrade(RecipeOutput consumer, IItemProvider output, TagKey<Item> circuitTag, TagKey<InfuseType> infusionType, int singleAlloyAmount, String basePath, String name) {
+        ItemStackChemicalToItemStackRecipeBuilder.metallurgicInfusing(
+              IngredientCreatorAccess.item().from(circuitTag),
+              IngredientCreatorAccess.infusionStack().from(infusionType, (int) (singleAlloyAmount * 1.5F)),
```

Comment:
I am not really a fan of this alternate recipe skipping a base input AND then on top of that lowering the amount of infuse type needed. I think we should instead have it use a bit more of the infuse type (say 3x or at least 2.5x as much), so that both recipes have their own merits and provide users an actual choice to make. Especially as for say higher tier circuits then this is removing the intermediary tier alloy infusing costs.

Reply:
I'm fine to increase the infusion amount, but the current system is more tedious than anything. Banging 6 ingots into a circuit doesn't make much sense, it's just a grind

Reply:
Yeah, which is why I am fine with adding an alternative way. I just feel like the alternative way should be more infuse type intensive than using the ingots. So both systems have their own merits and gives players a choice to make. I think 3x as much would be a good place to have it at.